dist/profile/templates/confluence/vhost.conf

```diff
@@ -6,6 +6,2813 @@ RewriteEngine on
+RewriteRule "^/display/BlameSubversion$" "https://plugins.jenkins.io/BlameSubversion" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/BlazeMeter\+Plugin$" "https://plugins.jenkins.io/BlazeMeterJenkinsPlugin" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/CFLint\+plugin$" "https://plugins.jenkins.io/CFLint" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/ColumnPack\+Plugin$" "https://plugins.jenkins.io/ColumnsPlugin" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/Custom\+History$" "https://plugins.jenkins.io/CustomHistory" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/DotCi\+Plugin$" "https://plugins.jenkins.io/DotCi" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/DotCi\+DockerPublish\+Plugin$" "https://plugins.jenkins.io/DotCi-DockerPublish" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/DotCi\+InstallPackages\+Plugin$" "https://plugins.jenkins.io/DotCi-InstallPackages" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/DotCi\+Plugins\+Starter\+Pack\+Plugin$" "https://plugins.jenkins.io/DotCi-Plugins-Starter-Pack" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/Exclusion\-Plugin$" "https://plugins.jenkins.io/Exclusion" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/Gatekeeper\+Plugin$" "https://plugins.jenkins.io/GatekeeperPlugin" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JDK\+Parameter\+Plugin$" "https://plugins.jenkins.io/JDK_Parameter_Plugin" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JiraTestResultReporter\-plugin$" "https://plugins.jenkins.io/JiraTestResultReporter" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/Matrix\+configuration\+sorter\+plugin$" "https://plugins.jenkins.io/Matrix-sorter-plugin" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/Office\+365\+Connector\+Plugin$" "https://plugins.jenkins.io/Office-365-Connector" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/Parameterized\+Remote\+Trigger\+Plugin$" "https://plugins.jenkins.io/Parameterized-Remote-Trigger" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/Priority\+Sorter\+Plugin$" "https://plugins.jenkins.io/PrioritySorter" [NC,L,QSA,R=301]
```

Comment:
Are you not missing /JENKINS/ between /display and plugin name?
An exampled `https://wiki.jenkins.io/display/JENKINS/Priority+Sorter+Plugin`